torch/csrc/utils/tensor_new.cpp

```diff
@@ -596,6 +597,65 @@ Tensor indexing_tensor_from_data(
   }
 }
 
+Tensor sparse_csr_tensor_ctor(c10::DispatchKey dispatch_key, at::ScalarType scalar_type, PyObject* args, PyObject* kwargs) {
```

Comment:
More hand written Python binding code for constructors makes me a sad panda. (Not sure how feasible this is to fix upstream)

Reply:
I don't think this should be fixed in this PR. Delegate to later? The COO constructor will also be need to be fixed in a similar manner.